src/doom/g_game.c

```diff
@@ -2052,8 +2052,10 @@ int G_VanillaVersionCode(void)
 {
     switch (gameversion)
     {
+        case exe_doom_1_1:
+            return 101;
         case exe_doom_1_2:
-            I_Error("Doom 1.2 does not have a version code!");
+            return 102;
```

Comment:
The version code is the first byte of demo files that indicates the version number. This shouldn't be changed - it's still true that versions <= 1.2 didn't have this in the header.

Reply:
Reverted in https://github.com/chocolate-doom/chocolate-doom/pull/1205/commits/77f49e5ea5aae45fe904306a9364018c0fc23b71 .